api/pkg/s3/objectdelete.go

```diff
@@ -15,21 +15,14 @@
 package s3
 
 import (
-	"net/http"
-	"strings"
-
 	"github.com/emicklei/go-restful"
 	log "github.com/sirupsen/logrus"
-	. "github.com/opensds/multi-cloud/s3/pkg/exception"
-	"github.com/opensds/multi-cloud/s3/proto"
-	"github.com/opensds/multi-cloud/api/pkg/common"
 )
 
 func (s *APIService) ObjectDelete(request *restful.Request, response *restful.Response) {
-	url := request.Request.URL
-	log.Infof("URL is %v", request.Request.URL.String())
-
-	bucketName := request.PathParameter("bucketName")
```

Comment:
does DELETE need a re-confirmation ? "Are you sure you want to delete" kind of thing ?

Reply:
@rhsakarpos the function of object delete is not finished yet. if we finish the object delete functions, it will introduce more changed files, can we fix this problem in next pr which contains object delete function? thanks.

Reply:
please remove the incomplete DELETE code from this PR in that case, thanks

Reply:
@rhsakarpos i have removed the codes from object delete function in that file. thanks a lot.